pkg/operation/botanist/addons.go

```diff
@@ -316,6 +303,7 @@ func (b *Botanist) generateCoreAddonsChart() (*chartrenderer.RenderedChart, erro
 		global          = map[string]interface{}{
 			"kubernetesVersion": b.Shoot.Info.Spec.Kubernetes.Version,
 			"podNetwork":        b.Shoot.Networks.Pods.String(),
+			"vpaEnabled":        false, // TODO: b.Shoot.WantsVerticalPodAutoscaler, once https://github.com/gardener/gardener-resource-manager/issues/58 is resolved
```

Comment:
I see that the issue is closed. Will this TODO be removed in this PR or later?

Reply:
Yep, thanks for reminding, will do it in this PR

Reply:
done, ptal